lxd/db/instances.go

```diff
@@ -393,9 +395,9 @@ func (c *Cluster) InstanceList(filter *InstanceFilter, instanceFunc func(inst In
 	return nil
 }
 
-// GetInstanceToNodeMap returns a map associating the name of each
+// GetProjectInstanceToNodeMap returns a map associating the project (key element 0) and name (key element 1) of each
 // instance in the given projects to the name of the node hosting the instance.
-func (c *ClusterTx) GetInstanceToNodeMap(projects []string, filter InstanceFilter) (map[string]string, error) {
+func (c *ClusterTx) GetProjectInstanceToNodeMap(projects []string, filter InstanceFilter) (map[[2]string]string, error) {
```

Comment:
@markylaing ah yes now I see. Thanks, good spot.

Reply:
This would have been a good contender for a separate commit. Don't worry now though.